commit 38330a97a5b678fa14d3e8b0e4e9012a2e7d6232
Author: Matt Flax <flatmax@flatmax.org>
Date:   Thu Aug 30 09:38:02 2018 +1000

    ASoC: cs4265: Add a S/PDIF enable switch
    
    commit f853d6b3ba345297974d877d8ed0f4a91eaca739 upstream.
    
    This patch adds a S/PDIF enable switch as a SOC_SINGLE.
    
    Signed-off-by: Matt Flax <flatmax@flatmax.org>
    Reviewed-by: Charles Keepax <ckeepax@opensource.cirrus.com>
    Signed-off-by: Mark Brown <broonie@kernel.org>

diff --git a/sound/soc/codecs/cs4265.c b/sound/soc/codecs/cs4265.c
index 407554175282..c90959d7adf6 100644
--- a/sound/soc/codecs/cs4265.c
+++ b/sound/soc/codecs/cs4265.c
@@ -154,6 +154,7 @@ static const struct snd_kcontrol_new cs4265_snd_controls[] = {
 	SOC_SINGLE("E to F Buffer Disable Switch", CS4265_SPDIF_CTL1,
 				6, 1, 0),
 	SOC_ENUM("C Data Access", cam_mode_enum),
+	SOC_SINGLE("SPDIF Switch", CS4265_SPDIF_CTL2, 5, 1, 1),
 	SOC_SINGLE("Validity Bit Control Switch", CS4265_SPDIF_CTL2,
 				3, 1, 0),
 	SOC_ENUM("SPDIF Mono/Stereo", spdif_mono_stereo_enum),
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1163_38330a97a5b678fa14d3e8b0e4e9012a2e7d6232.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1163_38330a97a5b678fa14d3e8b0e4e9012a2e7d6232.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
